### DISCUSSION TITLE
Is there a way to add a keybind for `up` and `down` actions?

### DISCUSSION THREAD

**Discussion by guillaumeLamanda** [nvim-tree/nvim-tree.lua#1811] [Q&A]
Hello here 👋 

I am configuring the plugin, but in the integrated doc I can't find any possibility to remap a key for the action `up` and `down`.  
I use a BEPO keyboard layout, so for me `r` si `up` and `n` is `down`. Do I miss something? 

Thanks for reading

**Comment by alex-courtis** [comment]
Unfortunately there is no action for up/down - it simply uses the regular vim navigation mappings.

Does r/n work for up/down in regular buffers? How have you mapped them?

Context: I'm using a dvorak layout, however I'm using the "regular" h/j/k/l without any dvorak specific configuration.

**Reply by guillaumeLamanda** [reply]
I have remapped `r` to `j` and `n` to `k` in a config file.  

I tried to use the `remove_keymaps` option to `true`, and verified nothing in the nvim tree mappings overlaps `n` and `r`, but it does not seem to work.

**Reply by alex-courtis** [reply]
That's tricky. h/j/k/l are not really mapped anywhere as they are vim builtins. We can only really remap to them then remap them elsewhere with all the "noremap" business.

How did you remap? Do those remappings work in other plugins like fugitive, bufexplorer or tagbar?
